index.js

```diff
@@ -27,16 +27,24 @@ var hooks = [
   'update'
 ];
 
+function findGitRoot() {
+  var gitRevParseTopLevel = exec('git rev-parse --show-toplevel');
+  if (gitRevParseTopLevel.code !== 0) {
+    console.error('guppy-cli needs a git repository to work with.');
+    return exit(1);
```

Comment:
Sorry for late response. Please check my latest commit. Does it now work out for you?